project/scripts/components/content/folded-list.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * This folds a folded list when the lists is more than 6 items long
+ * The sixth item is replaced by the text: "+ <number> meer"
+ *
+ * Once unfolded, the list cannot be folded again
+ */
+
+let delegate = require("delegate-events");
+var constants = window.HAN.constants;
+
+function foldList(e) {
+  // get list
+  let lists = document.getElementsByClassName("js-folded-list");
+  for (let i = 0; i < lists.length; i++) {
+    // if more than 6, fold
+    if (lists[i].childNodes.length > 6) {
+      for (
+        let j = lists[i].childElementCount;
+        j > constants.MAX_LENGTH_FOLDED_LISTS - 1;
+        j--
+      ) {
+        lists[i].childNodes[j - 1].classList.add(constants.HIDDEN_CLASS);
+      }
+      // at position 6, show "lees meer"
+      let el = document.createElement("li");
+      el.classList.add("folded-list__item");
+      el.classList.add("folded-list__item--readmore");
+      let content = document.createTextNode(
+        "+ " +
+          (lists[i].childElementCount - constants.MAX_LENGTH_FOLDED_LISTS + 1) +
+          " meer"
+      );
+      el.appendChild(content);
+      lists[i].appendChild(el);
+    }
+  }
+}
+
+function unfoldList(e) {
+  // get target list
+  let list = e.target.closest(".folded-list");
+  // show all items
+  for (let i = 0; i < list.childElementCount; i++) {
+    list.childNodes[i].classList.remove(constants.HIDDEN_CLASS);
+  }
+  // remove "lees meer"
+  e.target.parentNode.removeChild(e.target);
```

Comment:
hier moet de focus naar het eerste nieuwe list item toe gaan, ongeveer zoiets

```suggestion
  e.target.parentNode.removeChild(e.target);
  
  $list.querySelector(`li:nth-of-type${constants.MAX_LENGTH_FOLDED_LISTS + 1} a`).focus();
```